### PR TITLE
hck09

### PR DESCRIPTION
**Pull request by bendichter**

### CODE REVIEW

File: HCK09_2020_Remote/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+
+<img alt="Remote hackathon" src="../HCK08_2020_Remote/logo_brain_text_white_hor.png">
+
+## Welcome to the 8th NWB User Days Workshop!
```
Comment (oruebel) [inline HCK09_2020_Remote/README.md:4]:
```suggestion
## Welcome to the 9th NWB User Days Workshop!
```

File: HCK09_2020_Remote/README.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+## Dates and Location
+
+- **Dates:** September, 2020
+- **Location:** Your home!
+
+## Registration
+
+The workshop is open to the public. Please complete our [registration form](https://forms.gle/8LYAx743D4rEwSkT7) to attend the event. Anyone interested in NWB may request an invitation. 
+
```
Comment (oruebel) [inline HCK09_2020_Remote/README.md:25]: Need to make a new registration form and update the link before merging.

Comment (bendichter) [reply HCK09_2020_Remote/README.md:25]: already did

Comment (bendichter) [reply HCK09_2020_Remote/README.md:25]: but I guess the link isn't updated? Hold on, I'll fix this

**Pull request merged**